Rakefile

```diff
@@ -0,0 +1,29 @@
+LOCALEAPP_CONFIG      = ".localeapp/config.rb".freeze
+LOCALEAPP_CONFIG_DIR  = LOCALEAPP_CONFIG.pathmap("%d").freeze
+CI_SERVICE_ENV_KEY    = "TRAVIS".freeze
+
+
+directory LOCALEAPP_CONFIG_DIR
+
+file LOCALEAPP_CONFIG => LOCALEAPP_CONFIG_DIR do |t|
+  # FIXME: maybe it would be better to improve localeapp setup, or test env
+  # management, rather than creating a dummy config file.
+  #sh "touch #{t.name}" if ENV.key? CI_SERVICE_ENV_KEY
+  next unless ENV.key? CI_SERVICE_ENV_KEY
+  File.open t.name, "w" do |f|
+    f.write <<-eoh
+Localeapp.configure do |config|
+  config.api_key                    = 'DUMMY_KEY_FOR_TESTS'
+  config.translation_data_directory = 'locales'
+end
+    eoh
+  end
+end
```

Comment:
I agree if we were using it during the build, and if we wanted some
kind of acceptance / infrastructure testing (but then I think it would
belong to tests in `localeapp` gem).

In our case I'm not convinced, I think the build is unnecessarily
coupled to localeapp because of this:

https://github.com/alpinelab/website/blob/master/config.rb#L154
https://github.com/alpinelab/website/blob/master/Gemfile#L20


I see what you suggest, but wouldn't it be better with a specific
"dummy" standalone repository? And that kind of test would be to test
localeapp "infrastructure" (both the client and the service).